{"name": "test_demo2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == -1", "trace": "self = <test_mark标签.Test_Demo object at 0x000001A9F65BE7D0>\n\n    @pytest.mark.demo2\n    def test_demo2(self):\n        a = 5\n        b = -1\n>       assert a == b\nE       assert 5 == -1\n\ntest_mark标签.py:15: AssertionError"}, "start": 1701430294182, "stop": 1701430294182, "uuid": "ea4067c2-d75d-434b-a5fc-e3a55f88d70a", "historyId": "591faeae585796146de8703243f3657f", "testCaseId": "591faeae585796146de8703243f3657f", "fullName": "test_mark标签.Test_Demo#test_demo2", "labels": [{"name": "tag", "value": "demo2"}, {"name": "suite", "value": "test_mark标签"}, {"name": "subSuite", "value": "Test_Demo"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "14004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_mark标签"}]}